globus-requirements.txt

```diff
@@ -0,0 +1,6 @@
+# extra requirements for globus
+-r ./requirements.txt
+requests>=2.0.0,<3.0.0
+six==1.10.0
+python-jose
+pycrypto
```

Comment:
You should be okay to only depend on `python-jose`. `pycrypto` is just a dependency of `python-jose`.

I want to get around to making this doable via an extra, so you could do `globus_sdk[jwt]`, for JWT support, but the time to tinker with setuptools hasn't materialized.